### PR TITLE
fix: None value returned by Rating API

### PR DESCRIPTION
**Pull request by shadinaif**
fix: None value returned by Rating API

Before the fix: the API is crashing on tenants with no rating records

### CODE REVIEW

**Review by OmarIthawi — APPROVED** [review]

File: futurex_openedx_extensions/dashboard/statistics/courses.py [hunk 1]
```diff
@@ -87,47 +90,27 @@ def get_courses_ratings(
...
+            'feedbackcourse',
+            filter=Q(feedbackcourse__rating_content=rate_value)
+        ) for rate_value in range(1, 6)
+    })
+
+    return q_set.aggregate(
+        total_rating=Coalesce(Sum('rating_total'), 0),
+        courses_count=Coalesce(Count('id'), 0),
+        **{
+            f'rating_{rate_value}_count': Coalesce(Sum(f'course_rating_{rate_value}_count'), 0)
```
Comment (OmarIthawi) [inline futurex_openedx_extensions/dashboard/statistics/courses.py:113]: weired stuff going on here. super enterprisy SQL 😅

File: tests/test_dashboard/test_views.py [hunk 2]
```diff
@@ -619,6 +619,35 @@ def test_success(self):
...
+        self.login_user(self.staff_user)
+        with patch('futurex_openedx_extensions.dashboard.views.get_courses_ratings') as mocked_calc:
+            mocked_calc.return_value = {
+                'total_rating': 0,
+                'courses_count': 0,
+                'rating_1_count': 0,
+                'rating_2_count': 0,
+                'rating_3_count': 0,
+                'rating_4_count': 0,
+                'rating_5_count': 0,
```
Comment (OmarIthawi) [inline tests/test_dashboard/test_views.py:633]: nice, I like the result structure.

**Review by OmarIthawi — APPROVED** [review]
double approve, for the win!

File: futurex_openedx_extensions/dashboard/statistics/courses.py (hunk 1 above)
Comment (shadinaif) [reply futurex_openedx_extensions/dashboard/statistics/courses.py:113]: Overuse of DRY 😁

**Pull request merged**